docs/pages/setup/guides/joining-nodes-aws.mdx

```diff
@@ -1,22 +1,62 @@
 ---
 title: Joining Nodes in AWS
-description: How to join nodes and proxies on EC2
+description: How to join nodes and proxies running on AWS
 h1: Joining Node and Proxies in AWS
 ---
 
 This section will explain how to set up Teleport Nodes and Proxies to join your
-Teleport cluster with the EC2 join method. The EC2 join method enables you to
-allow Teleport nodes to join your Teleport cluster from EC2 instances in your
-AWS account without sharing any secrets.
+Teleport cluster without sharing any secrets when they are running in AWS.
+
+<Admonition type="note">
+You can still use secret tokens to join the cluster, see the
+[Getting Started](../../getting-started/linux-server.mdx#add-a-node-to-the-cluster)
+guide. The following AWS join methods are meant to make deployment simpler by
+eliminating the need to distribute secrets when your Nodes or Proxies are
+running in AWS.
+</Admonition>
+
+There are two join methods available depending on your use-case.
+
+The EC2 join method:
```

Comment:
Since the sentence fragment in each bullet has a different format, i.e., they don't all begin with "is," it might be clearer to turn this list into paragraphs. If that turns out to be a long stretch of text, we can break it up with two H2s: `## The EC2 join method` and `## The IAM join method`.